src/LaserDist/Properties/AssemblyInfo.cs

```diff
@@ -9,10 +9,10 @@
 [assembly: AssemblyConfiguration("")]
 [assembly: AssemblyCompany("")]
 [assembly: AssemblyProduct("LaserDist")]
-[assembly: AssemblyCopyright("Copyright ©2016,©2017 Steven Mading")]
+[assembly: AssemblyCopyright("Copyright ©2016,©2017,@2018 Steven Mading")]
```

Comment:
Does it matter that you have an @ rather than a ©?

Reply:
Not sure if it has any legal repurcussions, but I'll try to remember to fix it next time.